operate/common/src/main/java/io/camunda/operate/conditions/DatabaseInfo.java

```diff
@@ -61,6 +62,12 @@ public boolean isElasticsearchDb() {
     return isElasticsearch();
   }
 
+  @Override
+  public void destroy() {
+    // unset current so it can be reinitialized
+    current = null;
```

Comment:
❓️When do we reinitialize it? Only in tests?

Reply:
I think so, this is probably the first test the reuses the application.